Traceback (most recent call last):
  File "/GPUFS/sysu_zbzheng_1/sujzh3/mythril-develop/venv/bin/securify", line 33, in <module>
    sys.exit(load_entry_point('securify', 'console_scripts', 'securify')())
  File "/sec/securify/__main__.py", line 277, in main
    matches.extend(pattern.find_matches())
  File "/sec/securify/analyses/patterns/static/static_analysis_patterns.py", line 73, in find_matches
    static = analysis_context.static_analysis
  File "/sec/securify/analyses/analysis.py", line 88, in static_analysis
    self._static = self.config.static_analysis(self)
  File "/sec/securify/__main__.py", line 255, in <lambda>
    static_analysis=lambda t: static_analysis.analyze_cfg(t.cfg, **souffle_config),
  File "/sec/securify/analyses/analysis.py", line 81, in cfg
    self._cfg = self.config.cfg_compiler(self)
  File "/sec/securify/__main__.py", line 254, in <lambda>
    cfg_compiler=lambda t: solidity_cfg_compiler.compile_cfg(t.ast).cfg,
  File "/sec/securify/solidity/solidity_cfg_compiler.py", line 88, in compile_cfg
    cfg = ast_attr.cfg
  File "/sec/securify/grammar/attributes/evaluators/evaluator_demand_base.py", line 84, in __get__
    return self.__evaluator.evaluate(instance, self.__name)
  File "/sec/securify/grammar/attributes/evaluators/evaluator_demand_iterative.py", line 26, in evaluate
    rule_info = self[node].resolve_rule(attribute)
  File "/sec/securify/grammar/attributes/__init__.py", line 290, in resolve_rule
    node_dependencies = {c.node: self.resolve_node(rule_node, c.node, rule_node_production) for c in rule.arguments}
  File "/sec/securify/grammar/attributes/__init__.py", line 290, in <dictcomp>
    node_dependencies = {c.node: self.resolve_node(rule_node, c.node, rule_node_production) for c in rule.arguments}
  File "/sec/securify/grammar/__init__.py", line 616, in resolve_node
    raise AttributeError(f"Semantic rule references unavailable child node '{requested}'.")
AttributeError: Semantic rule references unavailable child node 'false_body'.
